### DISCUSSION TITLE
Where is my customized shortcut stored?

### DISCUSSION THREAD

**Discussion by LumenYoung** [pop-os/shell#1243] [Q&A] [answered]
Hello, everyone,

I was hit by a question today and I didn't find the answer myself: If I were to move to a new machine, is there a way to sync my customized shortcuts from the previous machine? I checked file in ~/.config/pop-shell, but the .json file in it didn't store such information. 

So I would like to ask, where are these configs stored in pop os, and how should I handle with it?

Regards,
Yang.

**Comment by jacobgkau** [comment, marked as answer]
They're stored in [gsettings](https://wiki.gnome.org/HowDoI/GSettings). You can run `gsettings list-keys org.gnome.shell.extensions.pop-shell` to get a list of keys, then `gsettings get org.gnome.shell.extensions.pop-shell key-name` to get a setting and `gsettings set org.gnome.shell.extensions.pop-shell key-name new-value` to set a setting. There's also a GUI program called `dconf-editor` you can use to browse these keys graphically.

I'm not aware of an easy way to "copy" the settings as files since they're part of a database, but creating a script to set them to what you want would be an easier alternative.

**Reply by LumenYoung** [reply]
Thank you for your answer! I will consider to use the script.

**Reply by huyz** [reply]
Is that still the case? `gsettings list-keys org.gnome.shell.extensions.pop-shell` returns nothing for me.

**Reply by jacobgkau** [reply]
> Is that still the case? `gsettings list-keys org.gnome.shell.extensions.pop-shell` returns nothing for me.

Yes, it is still the case. The command you pasted returns a list of keys for me on Pop!_OS 22.04:

```
jacob@serw12:~$ gsettings list-keys org.gnome.shell.extensions.pop-shell
activate-launcher
active-hint
active-hint-border-radius
column-size
focus-down
focus-left
focus-right
focus-up
fullscreen-launcher
gap-inner
gap-outer
hint-color-rgba
log-level
management-orientation
mouse-cursor-focus-location
mouse-cursor-follows-active-window
pop-monitor-down
pop-monitor-left
pop-monitor-right
pop-monitor-up
pop-workspace-down
pop-workspace-up
row-size
show-skip-taskbar
show-title
smart-gaps
snap-to-grid
tile-accept
tile-by-default
tile-enter
tile-move-down
tile-move-down-global
tile-move-left
tile-move-left-global
tile-move-right
tile-move-right-global
tile-move-up
tile-move-up-global
tile-orientation
tile-reject
tile-resize-down
tile-resize-left
tile-resize-right
tile-resize-up
tile-swap-down
tile-swap-left
tile-swap-right
tile-swap-up
toggle-floating
toggle-stacking
toggle-stacking-global
toggle-tiling
jacob@serw12:~$ 
```

**Reply by huyz** [reply]
I'm running pop shell on top of Ubuntu 22.04 with Gnome Wayland, but not pop OS. Could that be the difference?
